cmd/jaeger/es_config.yaml

```diff
@@ -0,0 +1,34 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+
+extensions:
+  jaeger_query:
+    trace_storage: es_main
+    trace_storage_archive: es_archive
+    ui_config: ./cmd/jaeger/config-ui.json
+
+  jaeger_storage:
+    elasticsearch:
+      es_main:
+        server_urls: http://127.0.0.1:9200
+        log_level: "error"
+      es_archive:
```

Comment:
is this a good example? archive storage implementation is identical to primary, so if you given them identical parameters they will be trying to write to the same data

Reply:
added `index_prefix` field in archive storage